lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -1021,7 +968,7 @@ WasmBinaryReader::ReadInlineName(uint32& length, uint32& nameLength)
     m_pc += nameLength;
     length += nameLength;
 
-    return CvtUtf8Str(rawName, nameLength);
+    return CvtUtf8Str(rawName, nameLength, &nameLength);
```

Comment:
This was causing decoded utf8 strings to have invalid length and thus we would allocate invalid buffer size